### PR TITLE
NEWS.md file added

### PR DESCRIPTION
**Pull request by insightsurge**
Added news.md file based on historic PRs and commits.

### Link to the issue #65 

Before submitting a PR please answer:
* Was it necessary to update README? Is README updated?  NA
* Are unit tests added for new logic? NA

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+-   README updated
+-   R version updated
+    -   windows: `3.6` to `4.1.2`
+    -   macOS: `4.0.2` to `4.1.2`
+    -   ubuntu 18.04: `3.6` to `4.1.2`
+
+# data.validator 0.1.5
+
+-   `assert_cols`, `assert_rows` and `assert_if` renamed to `validate_cols`, `validate_rows` and `validate_if`
+-   Documentation updated for newly added functions.
```
Comment (jakubnowicki) [inline NEWS.md:22]: Not worth mentioning.

File: NEWS.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    -   Connect workflow examples added
+-   pkgdown documentation updated
+
+# data.validator 0.1.6
+
+-   README updated
+-   R version updated
+    -   windows: `3.6` to `4.1.2`
+    -   macOS: `4.0.2` to `4.1.2`
+    -   ubuntu 18.04: `3.6` to `4.1.2`
```
Comment (jakubnowicki) [inline NEWS.md:17]: This is not a change that affects the code. Version 0.1.6 was pushed to CRAN only to restore `data.validator` (after one of its dependencies was removed) and there were no actual changes implemented. The changelog should reflect that.

File: NEWS.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# data.validator 0.2.0
+
+-   `validate_cols()` will use all columns in dataframe if no column is passed
+-   Added a defensive wrapper (`data.validator:::check_assertr_expression()`) to add evaluation error to regular validation errors
+-   New examples added:
+    -   Shiny App example demo example added
+    -   Usage examples added in README
+    -   Connect workflow examples added
+-   pkgdown documentation updated
```
Comment (jakubnowicki) [inline NEWS.md:9]: Not worth mentioning.

File: NEWS.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# data.validator 0.2.0
```
Comment (jakubnowicki) [inline NEWS.md:1]: Misses bugfix: https://github.com/Appsilon/data.validator/pull/48
MIsses fix: https://github.com/Appsilon/data.validator/pull/54

File: NEWS.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+-   `validate_cols()` will use all columns in dataframe if no column is passed
```
Comment (jakubnowicki) [inline NEWS.md:3]: `validate_rows` was also changed in the same PR: https://github.com/Appsilon/data.validator/pull/53

**Review by jakubnowicki — CHANGES_REQUESTED** [review]

File: NEWS.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+-   `validate_cols()` and `validate_rows()` will use all columns in dataframe if no column is passed
+
+-   Added a defensive wrapper (`data.validator:::check_assertr_expression()`) to add evaluation error to regular validation errors
+
+-   New examples added:
+
+    -   Shiny App example demo example added
+    -   Usage examples added in README
+    -   Connect workflow examples added
```
Comment (jakubnowicki) [inline NEWS.md:11]: Not worth mentioning.

File: NEWS.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+-   Removed deprecated `.data` calls in `dplyr::select()`
+
+-   `shiny.semantic::accordion()` bug fixes
+
+# data.validator 0.1.6
+
+-   Dependencies updated
```
Comment (jakubnowicki) [inline NEWS.md:19]: Please describe the purpose of this release, not a vague "dependencies update".

**Review by jakubnowicki — CHANGES_REQUESTED** [review]
Please remove empty lines between bullet points.
Please use single whitespace between "-" and the text (https://github.com/DavidAnson/markdownlint/blob/v0.28.2/doc/md030.md)
Please add the changelog to the pkgdown configuration - see [shiny.i18n](https://github.com/Appsilon/shiny.i18n/blob/main/pkgdown/_pkgdown.yml) for the reference.

File: NEWS.md (hunk 7 above)
Comment (insightsurge) [reply NEWS.md]: Only package dependency change I was able to find in #25 was related to ROxygen version upgrade.  So I've added this change in NewsMD file.

Comment (jakubnowicki) [reply NEWS.md]: As I wrote in my previous comment: Version 0.1.6 was pushed to CRAN only to restore data.validator (after one of its dependencies was removed) and there were __no actual changes__ implemented. The changelog should reflect that - you need to write that there were __no changes__ and describe why (in a super condensed way). Simply read the comments in the PR you linked here to learn more (shiny.semantic was down and all packages that depended on it were also removed from CRAN).

**Review by jakubnowicki — CHANGES_REQUESTED** [review]
Please check my reply to your comment.

File: NEWS.md [hunk 8]
```diff
@@ -0,0 +1,18 @@
+# data.validator 0.2.0
+
+- `validate_cols()` and `validate_rows()` will use all columns in dataframe if no column is passed
+- Added a defensive wrapper (`data.validator:::check_assertr_expression()`) to add evaluation error to regular validation errors
+- Removed deprecated `.data` calls in `dplyr::select()`
+- `shiny.semantic::accordion()` bug fixes
+
+# data.validator 0.1.6
+
+- No changes in this release. It was released to revive `data.validator` on CRAN, previously archived due to the archiving of `shiny.semantic` dependency.
```
Comment (jakubnowicki) [inline NEWS.md:10]:
```suggestion
No changes. Released to restore `data.validator` on CRAN, previously archived due to the archiving of `shiny.semantic` dependency.
```

**Review by jakubnowicki — CHANGES_REQUESTED** [review]
Please check my suggestion.
Also, the changelog misses [this PR](https://github.com/Appsilon/data.validator/pull/62) - please include it as the development version of the package:

# data.validator (development version)

**Comment by insightsurge** [comment]
> Please check my suggestion. Also, the changelog misses [this PR](https://github.com/Appsilon/data.validator/pull/62) - please include it as the development version of the package:
> 
> # data.validator (development version)

Should I mention the exact version (i.e., 0.2.0.9000) ?

**Comment by jakubnowicki** [comment]
> > Please check my suggestion. Also, the changelog misses [this PR](https://github.com/Appsilon/data.validator/pull/62) - please include it as the development version of the package:
> > # data.validator (development version)
> 
> Should I mention the exact version (i.e., 0.2.0.9000) ?

@insightsurge no, please use exactly what I posted: `# data.validator (development version)`

**Review by jakubnowicki — APPROVED** [review]
LGTM

**Pull request merged**